src/common/map/MapDirective.js

```diff
@@ -13,6 +13,47 @@
             zoom: '='
           },
           link: function(scope, element) {
+
+            function createVector(coor) {
+              var geojsonObject = {
+                'type': 'FeatureCollection',
+                'crs': {
+                  'type': 'name',
+                  'properties': {
+                    'name': 'EPSG:3857'
+                  }
+                },
+                'features': [{
+                  'type': 'Feature',
+                  'geometry': {
+                    'type': 'Polygon',
+                    'coordinates': [[
+                      ol.proj.transform([coor[0], coor[1]], 'EPSG:4326', 'EPSG:3857'),
```

Comment:
Hi @milafrerichs , the idea is have a helper for the coordinates or an external json with the information? I'm a little confused.